### PR TITLE
create login

### PR DESCRIPTION
**Pull request by YuviQP**
-login
-styles login page

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -21,8 +21,9 @@ import MainPage from './pages/HomePage';
 import NewTopic from './pages/NewTopic';
 import NewTopicExample from 'pages/NewTopicExample';
 import NewEntry from 'pages/NewEntry';
-import RegisterAccount from 'views/pages/authentication/register_account';
+import RegisterAccount from 'views/pages/authentication/register_account/index';
```
Comment (PDFAtauchi) [inline src/index.js:24]: Update the index.js to be RegisterAccount.js

File: src/views/pages/authentication/login-account/login-account.js [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+    const [hasErrors, setHasErrors] = useState(true);
+
+    const handleToggleShowPassword = () => {
+        setShowPassword(!showPassword);
+    };
+
+    const handleChange = (e) => {
+        const { name, value } = e.target;
+        setFormData((prev) => {
+            if (name === 'password') {
```
Comment (PDFAtauchi) [inline src/views/pages/authentication/login-account/login-account.js:38]: if (name === 'password') {
                return {
                    ...prev,
                    [name]: value,
                    confirmPassword: '',
                }
            }
            
            this code is not needed

**Review by PDFAtauchi — CHANGES_REQUESTED** [review]

File: src/views/pages/authentication/login-account/login-account.js (hunk 2 above)
Comment (YuviQP) [reply src/views/pages/authentication/login-account/login-account.js]: Done

**Comment by PDFAtauchi** [comment]
LGTM! 🚀

**Pull request merged**